wordpress-mobile/WordPress-iOS
# Add ability to remove site followers/viewers

Fixes #5673 

This PR adds the ability to remove followers and viewers from a given site. The alert text is taken directly from Calypso.

![f](https://cloud.githubusercontent.com/assets/154014/24215938/67cdbcc2-0f08-11e7-802f-12121d9c88d8.png)
![v](https://cloud.githubusercontent.com/assets/154014/24215940/6ad203d8-0f08-11e7-940a-6521bee2c03b.png)
![delete_followers](https://cloud.githubusercontent.com/assets/154014/24215942/6d8274be-0f08-11e7-824a-f0059332017a.gif)

**To test:**

*Follower Removal:*
1. With a test account, follow a test site
2. In the WPiOS people management screen, navigate to the site's followers
3. Select your test follower from step 1. 
4. On the person screen, verify the remove button is visible
5. Tap remove, verify the alert view appears with the appropriate text (compare to Calypso if desired)
6. Tap remove on the alert view. Verify you are taken back to the followers table and the person is removed. (Refresh the table to reload and verify again)
7. Verify the follower is removed from the site in Calypso

*Viewer Removal:*
1. Add a test account (or more) as a viewer to a private site
2. In the WPiOS people management screen, navigate to the site's viewers
3. Select your test viewer from step 1. 
4. On the person screen, verify the remove button is visible
5. Tap remove, verify the alert view appears with the appropriate text (compare to Calypso if desired)
6. Tap remove on the alert view. Verify you are taken back to the viewers table and the person is removed. (Refresh the table to reload and verify again)
7. Verify the viewer is removed from the site in Calypso

Needs review: @jleandroperez would you mind reviewing this since you imp'ed a lot of people management?
